### ISSUE TITLE
Add example dataset so examples can run in pkg

### ISSUE THREAD

**Issue by sckott** [getwilds/fh.wdlR#13]

**Comment by sckott** [comment]
Talk to Sita and Amy and see what they want to do here

**Comment by tefirman** [comment]
Looks like this already exists [here](https://github.com/getwilds/rcromwell/tree/main/inst/examples), but maybe we can point to the WILDS WDL workflows for more examples? Maybe we can add them into the workflow submission section of the "Getting Started" vignette in the documentation?

**Comment by sckott** [comment]
Yes there are some examples, but I need more. There's a TON of untested code paths in this package, mostly in the parsing of response data. Since I don't know much about WDL/etc. I need Sita/Amy/others to get me example files that will test those paths. So I just need to get together with someone willing to help and then we can get it sorted pretty quick in theory
